keel-web/src/test/resources/examples/alb-example.yml

```diff
@@ -1,5 +1,5 @@
 ---
-kind: ec2/application-load-balancer@v1
+kind: ec2/application-load-balancer@v1.1
```

Comment:
Not a blocker, but would be nice to add those properties that caught us in the first place to this example.